resources/metadata/npmExecuteLint.yaml

```diff
@@ -0,0 +1,39 @@
+metadata:
+  name: npmExecuteLint
```

Comment:
I think we need the same step alias as in npmExecuteScripts to make the defaultNpmRegistry by taken from the npm configuration.